src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -5,51 +5,38 @@ import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 import CallToActionWidget from 'platform/site-wide/cta-widget';
 import WizardContainer from '../../wizard/containers/WizardContainer';
-import {
-  WIZARD_STATUS_NOT_STARTED,
-  WIZARD_STATUS_COMPLETE,
-} from 'applications/static-pages/wizard';
+import { WIZARD_STATUS_COMPLETE } from 'applications/static-pages/wizard';
 import { connect } from 'react-redux';
 import { showEduBenefits0994Wizard } from '../../selectors/educationWizard';
 
 export class IntroductionPage extends React.Component {
-  state = {
-    wizardStatus:
-      sessionStorage.getItem('wizardStatus') || WIZARD_STATUS_NOT_STARTED,
-  };
+  state = { status: sessionStorage.getItem('wizardStatus') };
```

Comment:
I was thinking it might be better to have the `wizardStatus` key set as a constant within the static-pages/wizard file.

Reply:
yep this value originates from the static-pages/wizard file [here](https://github.com/department-of-veterans-affairs/vets-website/blob/31bd56c30f4926b72dae7488c1d3b9907963dbbf/src/applications/edu-benefits/wizard/containers/WizardContainer.jsx#L75 ) and is being passed to sessionStorage to allow the changes to persist is that what ya mean?

Reply:
I meant that within the `static-pages/wizard` file, we should include this:

```js
export const WIZARD_STATUS = 'wizardStatus`;
```

that way we have a single source for the session storage item key.

Reply:
ahh i gotcha yep good call ill add that thanks